docs/en/latest/admin-api.md

```diff
@@ -21,75 +21,68 @@ title: Admin API
-| GET    | /apisix/admin/routes/{id}        | NULL         | Fetch resource                                                                                                                                                                                                                                                                                                    |
-| PUT    | /apisix/admin/routes/{id}        | {...}        | Create resource by ID                                                                                                                                                                                                                                                                                             |
-| POST   | /apisix/admin/routes             | {...}        | Create resource, and ID is generated by server                                                                                                                                                                                                                                                                    |
-| DELETE | /apisix/admin/routes/{id}        | NULL         | Remove resource                                                                                                                                                                                                                                                                                                   |
-| PATCH  | /apisix/admin/routes/{id}        | {...}        | Standard PATCH. Update some attributes of the existing Route, and other attributes not involved will remain as they are; if you want to delete an attribute, set the value of the attribute Set to null to delete; especially, when the value of the attribute is an array, the attribute will be updated in full |
-| PATCH  | /apisix/admin/routes/{id}/{path} | {...}        | SubPath PATCH, specify the attribute of Route to be updated through {path}, update the value of this attribute in full, and other attributes that are not involved will remain as they are. The difference between the two PATCH can refer to the following examples                                              |
+| Method | Request URI                      | Request Body | Description                                                                                                                   |
+| ------ | -------------------------------- | ------------ | ----------------------------------------------------------------------------------------------------------------------------- |
+| GET    | /apisix/admin/routes             | NULL         | Fetches a list of all configured Routes. list                                                                                 |
```

Comment:
The ending `list` is redundant?

Reply:
@moonming Using a list keeps it consistent with other items in the page even though there is only one item. It was a single item list before as well and I did not change it.

Should I move it out of the list or keep it as it is?

Reply:
I would prefer removing the `list` currently because your description is clearer. 😉

Reply:
Oh. I misread the message and did not see the actual `list` word. I will remove that.